### PR TITLE
added tooltips to event timestamps

### PR DESCRIPTION
**Pull request by DanielUche**
This PR adds 
1. Timestamp tooltips to the events table.

### CODE REVIEW

File: src/app/frontend/common/components/resourcelist/event/template.html [hunk 1]
```diff
@@ -62,14 +62,20 @@
 
       <ng-container [matColumnDef]="getDisplayColumns()[5]">
         <mat-header-cell *matHeaderCellDef>First Seen</mat-header-cell>
-        <mat-cell *matCellDef="let event">
+        <mat-cell *matCellDef="let event"
+          matTooltip="{{event.firstSeen | date:'medium'}}"
+          [matTooltipPosition]="top"
```
Comment (jeefy) [inline src/app/frontend/common/components/resourcelist/event/template.html]: This should be `matTooltipPosition="above"`

Enclosing the attribute in [] tries binding the value within the angular component and that's not needed.

"top" is not a valid value for this either, it's "above" -- See https://material.angular.io/components/tooltip/overview

File: src/app/frontend/common/components/resourcelist/event/template.html [hunk 2]
```diff
@@ -62,14 +62,20 @@
...
         </mat-cell>
       </ng-container>
 
       <ng-container [matColumnDef]="getDisplayColumns()[6]">
         <mat-header-cell *matHeaderCellDef>Last Seen</mat-header-cell>
-        <mat-cell *matCellDef="let event">
+        <mat-cell *matCellDef="let event"
+          matTooltip="{{event.lastSeen | date:'medium'}}"
+          [matTooltipPosition]="top"
+        >
```
Comment (maciaszczykm) [inline src/app/frontend/common/components/resourcelist/event/template.html]: Formatting is broken here.

Comment (DanielUche) [reply src/app/frontend/common/components/resourcelist/event/template.html]: will fix and revert

File: src/app/frontend/common/components/resourcelist/event/template.html (hunk 1 above)
Comment (DanielUche) [reply src/app/frontend/common/components/resourcelist/event/template.html]: will fix and reverse thanks

File: src/app/frontend/common/components/resourcelist/event/template.html [hunk 3]
```diff
@@ -62,14 +62,18 @@
...
           {{event.firstSeen | date}}
         </mat-cell>
       </ng-container>
 
       <ng-container [matColumnDef]="getDisplayColumns()[6]">
         <mat-header-cell *matHeaderCellDef>Last Seen</mat-header-cell>
-        <mat-cell *matCellDef="let event">
+        <mat-cell *matCellDef="let event"
+          matTooltip="{{event.lastSeen | date:'medium'}}"
+          matTooltipPosition="above">
```
Comment (maciaszczykm) [inline src/app/frontend/common/components/resourcelist/event/template.html]: We keep the default position in other places. Remove it and we can merge.

Comment (jeefy) [reply src/app/frontend/common/components/resourcelist/event/template.html]: @DanielUche Ping! Make that small change and we can merge this in. :) Thanks!

**Comment by maciaszczykm** [comment]
Applied the changes.

/lgtm
/approve

**Comment by eloyekunle** [comment]
I don't think the issue completely fixes #3631, although it is a step in the right direction.
The goal is to add these tooltips everywhere a time is displayed on the dashboard.

**Comment by maciaszczykm** [comment]
There is a formatting issue. I will close it as @DanielUche is not responding. Please reopen if you are willing to apply required changes, then we can merge.

/close

**Pull request closed without merging**